sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/config/ThroughputGlobalControlGroup.java

```diff
@@ -0,0 +1,47 @@
+package com.azure.cosmos.implementation.throughputControl.config;
+
+import com.azure.cosmos.CosmosAsyncContainer;
+
+import java.time.Duration;
+
+import static com.azure.cosmos.implementation.guava25.base.Preconditions.checkNotNull;
+
+public class ThroughputGlobalControlGroup extends ThroughputControlGroupInternal {
+    private static final Duration DEFAULT_CONTROL_ITEM_RENEW_INTERVAL = Duration.ofSeconds(10);
```

Comment:
My gut feeling is that this is too aggressive. But definitely don't block - this is something we need to test anyway. Just a thought - maybe using something like 10 seconds for the first 1 - 10 minutes and then increase it to let's say 30-60 seconds?

Reply:
But maybe that just over-complicates things. Never mind...

Reply:
Control item renew interval is used to control how often the ru will reload balanced between clients. And this is a value the customer can control, so using different value based on the time may not be an option here. But can discuss more  based on the test results.